apps-rendering/src/client/nativeCommunication.ts

```diff
@@ -233,20 +233,34 @@ function reportNativeElementPositionChanges(): void {
 		attributeFilter: ['style'],
 	};
 	const callback = function (): void {
-		const currentAdSlots = getAdSlots();
-		const currentVideoSlots = getVideoSlots();
+		try {
+			const currentAdSlots = getAdSlots();
 
-		if (positionChanged(currentAdSlots, adSlots)) {
-			adSlots = currentAdSlots;
-			void commercialClient.updateAdverts(currentAdSlots);
+			if (positionChanged(currentAdSlots, adSlots)) {
+				adSlots = currentAdSlots;
+				void commercialClient.updateAdverts(currentAdSlots);
+			}
+		} catch (ex) {
+			logger.error(`Exception updating ads`);
```

Comment:
I'd suggest replacing these with single quotes (`'`)? The backtick indicates a template string [^1], which is typically used for string interpolation. In this case you probably just need an ordinary string literal as you're not making use of interpolation (or multi-line strings).

[^1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals